GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -186,11 +180,9 @@ public static bool TryCreateAzDevOrgUrlFromPackageFeedUrl(string packageFeedUrl,
         /// <param name="src">The unprocessed string to use as arguments to an install command</param>
         /// <param name="installationId">A unique installer ID to replace the installer_id token with.</param>
         /// <returns>The argument string with tokens replaced.</returns>
-        public static string ReplaceArgTokens(string src, string installationId)
+        public static string ReplaceArgTokens(string src, string installationId, string logsDirectory)
         {
-            string logDirectory = ProductUpgraderInfo.GetLogDirectoryPath();
-
-            string dst = src.Replace(NuGetUpgrader.replacementToken(InstallActionInfo.ManifestEntryLogDirectoryToken), logDirectory)
+            string dst = src.Replace(NuGetUpgrader.replacementToken(InstallActionInfo.ManifestEntryLogDirectoryToken), logsDirectory)

```

Comment:
Not part of this PR, but what does dst stand for?

Reply:
src -> dst : source -> destination